### DISCUSSION TITLE
Support ROS Humble Hawksbill on Ubuntu 22.04?

### DISCUSSION THREAD

**Discussion by rkburns** [iRobotEducation/create3_docs#70] [Questions] [answered]
I'd like to move to this (very) recent release. Is there an ETA for support?

**Comment by shamlian** [comment]
Thank you for starting this discussion! We are excited about the recent release of ROS 2 Humble Hawksbill and are working on making this upgrade available for the Create 3. While we cannot provide a definitive timeline right now, we are aiming to have a public build available by August, with documentation updates to follow soon afterward. I will update this topic when the build is available.

**Reply by gbr1** [reply]
Hi @shamlian any news on humble release? Thank you in advance!

**Reply by justinIRBT** [reply]
We will get back to you next week on this.  We are still working on it internally.

**Reply by shamlian** [reply]
Sorry we missed August; we've had an exceptional month. It is now my expectation that we will have a public beta within a few weeks.

**Reply by JonathanNash21** [reply]
Hi, I'd love to be a part of this when it releases!

**Reply by shamlian** [reply]
I (or another maintainer) will post a new top level comment on this thread when the release is available, as well as updating the releases list on our docs site. If you (collectively) are reading this and want to be notified, please click the subscription button in the right menu bar of this page.

**Reply by shamlian** [reply]
Apologies if this is a double post: the Humble beta release is ready. Release notes (and a link to the download) can be found [here](https://iroboteducation.github.io/create3_docs/releases/h_0_0/). Please note that iRobot® Education Bluetooth protocol mode is not supported in this release. Please note also that ros-humble-irobot-create-msgs is also available as an installable package via apt as of this morning. Installation instructions for Ubuntu 22.04 can be found [here](https://iroboteducation.github.io/create3_docs/setup/ubuntu2204/).

**Comment by shamlian** [comment, marked as answer]
The Humble beta release is ready. Release notes (and a link to the download) can be found [here](https://iroboteducation.github.io/create3_docs/releases/h_0_0/). Please note that iRobot® Education Bluetooth protocol mode is not supported in this release. Please note also that `ros-humble-irobot-create-msgs` is also available as an installable package via `apt` as of this morning. Installation instructions for Ubuntu 22.04 can be found [here](https://iroboteducation.github.io/create3_docs/setup/ubuntu2204/).

**Comment by rickarmstrong** [comment]
To anyone who's tried the beta release: how's it going? I'm thinking about making the leap, and I'm curious to hear others' experiences. Successes? Problems?
